src/components/AvatarGroup/AvatarGroup.module.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/components/Avatar/Avatar.scss";
+@import "../../styles/themes";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  position: relative;
+}
+
+.avatarContainer {
+  margin-left: -8px;
+  // TODO customize spacing
+  :hover {
+    z-index: 2;
+  }
+}
+
+.counterContainer {
+  background: var(--primary-background-hover-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: -8px;
+  z-index: 1;
+
+  &:focus {
```

Comment:
I think u looking for focus-visible. because if the focus is not visible we don't want to display the ring. please see hoe we implement this in other components and I will explain it to you when we will talk (there is a reason why using a class name + :focus-visible)